### DISCUSSION TITLE
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module

### DISCUSSION THREAD

**Discussion by lili21** [remix-run/remix#3046] [Q&A] [answered]
after upgrade to 1.4.3, `remix dev` throw errors

```
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module : /Users/lili/github/shadowing/node_modules/sort-package-json/index.js
```

**Comment by lili21** [comment]
I upgraded it from version 1.2.3, and the 1.2.3 version works as expected.

**Reply by silvenon** [reply]
What code causes this? Could you tell us something about your setup?

**Comment by MichaelDeBoey** [comment, marked as answer]
Duplicate of #3044

This is fixed now that [`sort-package-json`](https://github.com/keithamus/sort-package-json) released `v1.56.1`
